client/src/components/ReviewAnswers.js

```diff
@@ -19,31 +96,56 @@ const ReviewAnswers = ({
   numOfQuestions,
 }) => {
   return (
-    <div>
+    <Container>
       <p>
         Your score is: {score} / {numOfQuestions}
       </p>
-      {questions.map(question => (
-        <Card key={question._id} className="mb-4">
-          <Card.Body>
-            <Card.Title>{question.text}</Card.Title>
-            <ul className="mb-0">
-              {question.answers.map(answer => (
-                <li
-                  key={answer._id}
-                  className={getAnswersClass(question, userAnswers, answer)}
-                >
-                  {answer.text}
-                </li>
-              ))}
-            </ul>
-          </Card.Body>
-        </Card>
-      ))}
-      <Button className="mb-4" variant="primary" href={quizName}>
-        Try again
-      </Button>
-    </div>
+      <Form>
+        <fieldset disabled>
+          {questions.map(question => (
+            <Card key={question._id} className="mb-4">
+              <Card.Header as="h5">{question.text}</Card.Header>
+              <Card.Body className="bg-white">
+                <Form.Group className="bg-light" key={question._id}>
```

Comment:
The `Form.Group` component is used to group inputs. Since there's no input inside it then it should be removed. Also, the `key` is only necessary in the first line of the `map` `return` so it is not necessary here